### DISCUSSION TITLE
成績上傳錯誤

### DISCUSSION THREAD

**Discussion by WhiteCloud0106** [Lost-MSth/Arcaea-server#145] [Q&A] [answered]
又是我嘿嘿，這次是成績上傳的時候顯示錯誤提示
![419688458_2722707071213880_1052271218600461999_n](https://github.com/Lost-MSth/Arcaea-server/assets/94116682/9bf5b3bf-1935-4b2b-bc48-e6a146c69e5a)
我去看了一下後台顯示這樣
![82_9RJYXZSIL%DIK)65 I69](https://github.com/Lost-MSth/Arcaea-server/assets/94116682/5552f661-8f0c-4c62-aa2d-85e9b8783eec)
這是發生了甚麼問題?

**Comment by Lost-MSth** [comment]
确保没有设定 `ALLOW_SCORE_WITH_NO_SONG = False`（默认是 `True`），确保服务端运行后没有修改过歌曲数据文件（否则请刷新 hash）

**Reply by WhiteCloud0106** [reply]
Allow score with no song要在哪裡看呢?

**Reply by Lost-MSth** [reply]
没在 `config.py` 里看到，就是默认值，没问题

**Reply by WhiteCloud0106** [reply]
的確沒看到，然後剛剛在網頁端刷新hash之後重啟伺服器但還是上傳失敗
這會不會跟他是世界模式曲目有關
他現在PST、PRS、FTR難度都無法上傳，但BYD可以

**Reply by Lost-MSth** [reply, marked as answer]
我不知道了，这个问题只有分数有问题、客户端服务端谱面不同之类的会触发（客户端重进可以拉取服务端 hash 数据）

实在不行就把校验去掉吧，找到报错里写的那个判断 `score.py` 的 101 行，注释掉

**Reply by WhiteCloud0106** [reply]
譜面不同😰好像譜面確實不同的樣子欸
不過因為這是不用下載的曲子我就沒更動了
原來不需下載的曲子也要放進伺服器嗎

**Reply by Lost-MSth** [reply]
不是，不需下载的就不要放服务器了，会被校验

**Reply by WhiteCloud0106** [reply]
原來是這樣啊，感謝你
